<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background:rgb(255, 85, 0)
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget
{
	background:rgb(255, 85, 0)
}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>501</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>35</pointsize>
      <weight>7</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color: white;
	font: 63 35pt &quot;Yu Gothic UI Semibold&quot;;
}</string>
    </property>
    <property name="text">
     <string>Register Your Account</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	border: 2px solid rgb(0, 0, 0);        /* Blue border */
    border-radius: 10px;             /* Rounded corners */
    padding: 5px;                    /* Inner padding */
    font-size: 14px;                 /* Text size */
    color: #2c3e50;                  /* Text color */
    background-color: #ecf0f1;
}</string>
    </property>
    <property name="text">
     <string>First Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>110</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	border: 2px solid rgb(0, 0, 0);        /* Blue border */
    border-radius: 10px;             /* Rounded corners */
    padding: 5px;                    /* Inner padding */
    font-size: 14px;                 /* Text size */
    color: #2c3e50;                  /* Text color */
    background-color: #ecf0f1;
}</string>
    </property>
    <property name="text">
     <string>Last Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	border: 2px solid rgb(0, 0, 0);        /* Blue border */
    border-radius: 10px;             /* Rounded corners */
    padding: 5px;                    /* Inner padding */
    font-size: 14px;                 /* Text size */
    color: #2c3e50;                  /* Text color */
    background-color: #ecf0f1;
}</string>
    </property>
    <property name="text">
     <string>Email Address</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	border: 2px solid rgb(0, 0, 0);        /* Blue border */
    border-radius: 10px;             /* Rounded corners */
    padding: 5px;                    /* Inner padding */
    font-size: 14px;                 /* Text size */
    color: #2c3e50;                  /* Text color */
    background-color: #ecf0f1;
}</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	border: 2px solid rgb(0, 0, 0);        /* Blue border */
    border-radius: 10px;             /* Rounded corners */
    padding: 5px;                    /* Inner padding */
    font-size: 14px;                 /* Text size */
    color: #2c3e50;                  /* Text color */
    background-color: #ecf0f1;
}</string>
    </property>
    <property name="text">
     <string>Contact Number</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox
{
	border: 2px solid rgb(0, 0, 0);        /* Blue border */
    border-radius: 10px;             /* Rounded corners */
    padding: 5px;                    /* Inner padding */
    font-size: 14px;                 /* Text size */
    color: #2c3e50;                  /* Text color */
    background-color: #ecf0f1;
}</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentText">
     <string>Karachi</string>
    </property>
    <item>
     <property name="text">
      <string>Karachi</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Lahore</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Islamabad</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>420</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	border: 2px solid rgb(0, 0, 0);        /* Blue border */
    border-radius: 10px;             /* Rounded corners */
    padding: 5px;                    /* Inner padding */
    font-size: 14px;                 /* Text size */
    color: #2c3e50;                  /* Text color */
    background-color: #ecf0f1;
}</string>
    </property>
    <property name="text">
     <string>Delivery Address</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>480</y>
      <width>194</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: white;       /* Blue background */
    color: black;                    /* White text */
    border: 2px solid rgb(0, 0, 0);       /* Darker blue border */
    border-radius: 12px;             /* Rounded corners */
    font-size: 20px;                 /* Larger font */
    padding: 10px 20px;              /* Add some padding */
    min-width: 150px; 
}</string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
